kirby/api/queue.py

```diff
@@ -48,7 +48,7 @@ def init_kafka(self):
                 self.name,
                 group_id=getenv("KIRBY_SUPERVISOR_GROUP_ID", type=str),
                 enable_auto_commit=True,
-                value_deserializer=msgpack.loads,
+                value_deserializer=(lambda x: msgpack.loads(x, raw=False)),
```

Comment:
what about using `kirby_value_deserializer` then?

Reply:
Now using deserializer and serializer in Queue